hieradata/node/lsstcam-pcs01.cp.lsst.org.yaml

```diff
@@ -0,0 +1,37 @@
+---
+nm::connections:
+  enp1s0:
+    content:
+      connection:
+        id: "enp1s0"
+        uuid: "03da7500-2101-c722-2438-d0d006c28c73"
+        type: "ethernet"
+        interface-name: "enp1s0"
+      ethernet: {}
+      ipv4:
+        address1: "139.229.175.94/26,139.229.175.126"
+        dns: "139.229.160.53;139.229.160.54;139.229.160.55;"
+        dns-search: "cp.lsst.org;"
+        method: "manual"
+      ipv6:
+        method: "disabled"
+      proxy: {}
+  enp2s0:
+    content:
+      connection:
+        id: "enp2s0"
+        uuid: "edb4ebb1-6d05-427b-b6ea-d483bdf32ec7"
+        type: "ethernet"
+        interface-name: "enp2s0"
+      ethernet: {}
+      ipv4:
+        method: "manual"
+        address1: "192.168.1.1/24"
```

Comment:
Does this conflict with lsstcam-ccobwb or is this a different network segment?

https://ipam.cp.lsst.org/index.php?page=subnets&section=1&subnetId=144&sPage=address-details&ipaddrid=5406

Reply:
@jhoblitt It shouldn't. From what I understand this is a private ip that will only talk to the other chiller units. It won't be connected to our network at all.